superset/viz.py

```diff
@@ -584,8 +584,12 @@ def get_data(self, df):
         # Display metrics side by side with each column
         if self.form_data.get('combine_metric'):
             df = df.stack(0).unstack()
+        parsed_columns = list(map(
```

Comment:
Not sure if this code is needed / wanted but maps in python are a bit convoluted. What about using list comprehension?

```
to_list = lambda x: [six.text_type(z) for z in x] if isinstance(x, tuple) else [x]
parsed_columns = [to_list(x) for x in df.columns.values]
```

Reply:
Yeah, OK for me. Guido don't like map/reduce/filter, LOL.